src/utils/exerciseServer.ts

```diff
@@ -38,6 +38,7 @@ export async function exerciseServer(testDir: string, replayScriptPath: string,
 
     const oldCwd = process.cwd();
     const replayScriptHandle = await fs.promises.open(replayScriptPath, "w");
+
```

Comment:
Maybe just revert the whole file?  `git checkout -f main -- src/utils/exerciseServer.ts` should work.